typescript/phoenix-sdk/src/instructions/CancelAllOrders.ts

```diff
@@ -5,25 +5,25 @@
  * See: https://github.com/metaplex-foundation/solita
  */
 
-import * as splToken from "@solana/spl-token";
-import * as beet from "@metaplex-foundation/beet";
-import * as web3 from "@solana/web3.js";
+import * as splToken from '@solana/spl-token'
```

Comment:
why did we switch from " to '? i don't care much either way but let's just make sure we are consistent